changelogs/fragments/26-remove_tls_requirements.yml

```
@@ -0,0 +1,7 @@
+bugfixes:
+- mysql_user - Correct procedure to check existing TLS requirements.
+- mysql_user - Minor syntax fixes.
+improvements:
+- mysql_user - Added tests to verify that TLS requirements are removed with an empty `tls_requires` parameter.
```

```suggestion
- mysql_user - added tests to verify that TLS requirements are removed with an empty ``tls_requires`` parameter (https:// link to the reported issue).
```

@Jorge-Rodriguez thanks! please, change the other entry as well, https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to

---

we should use `option` instead of parameter / arguments according to the doc best practices.

---

I think I've used parameter in a number of places, let me review that.

---

it's just a best practice in Ansible. i think we should follow it as the others practices.

---

If it's interesting to you https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#style-guide and also the several references at the page's bottom